### ISSUE TITLE
sql_escape_host_variables false positive

### ISSUE THREAD

**Issue by flaiker** [abaplint/abaplint#862]
> Escape SQL host variables (sql_escape_host_variables)

```abap
SELECT COUNT(*) FROM usr01 WHERE bname = @iv_user_name.
```

**Comment by larshp** [comment]
thanks 👍
